crates/biome_cli/src/commands/rage.rs

```diff
@@ -215,6 +285,54 @@ impl Display for RageConfiguration<'_, '_> {
     }
 }
 
+struct RageConfigurationLintRules<'a, T>(&'a str, T);
+
+impl<T> Display for RageConfigurationLintRules<'_, T>
+where
+    T: Serialize,
+{
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> io::Result<()> {
+        let rules_str = self.0;
+        write!(fmt, "  {rules_str}:")?;
+
+        let rule_json_str = serde_json::to_string(&self.1)
+            .map_err(|_| io::Error::new(io::ErrorKind::Other, "Failed to serialize"))?;
+        let category_to_rules: HashMap<String, Value> =
```

Comment:
What you call "category", is actually called **group**. We already have the concept of category for rules, but it's not what you think. `a11y`, `correctness`, etc. are called **groups**. Would mind changing the name to avoid a misunderstanding?

Reply:
Fixed 9a81817